python/tk_desktop/command_panel/recent_button.py

```diff
@@ -97,16 +100,3 @@ def command_name(self):
         Name of the command.
         """
         return self._command_name
-
-    def sizeHint(self):
```

Comment:
This doesn't get reevaluated when the window size changes, so basing the width off the parent widget size meant the size got stuck and didn't grow when expanding the window.
Having the items expand as the window expands removes the need for this method to be implented.